includes/class-wp-job-manager-post-types.php

```diff
@@ -888,32 +895,135 @@ public function set_expiry( $post ) {
 			return;
 		}
 
-		// See if it is already set.
-		if ( metadata_exists( 'post', $post->ID, '_job_expires' ) ) {
-			$expires = get_post_meta( $post->ID, '_job_expires', true );
-			if ( $expires && strtotime( $expires ) < current_time( 'timestamp' ) ) {
-				update_post_meta( $post->ID, '_job_expires', '' );
-			}
+		$expiration_date = $this->get_job_expiration( $post );
```

Comment:
Do we even need this? Couldn't we just use `$this->has_job_expired( $post )` on its own?

Reply:
Yup bd86cf7a86dbb7b2f228495455b2a188727b4f72